content/docs/Configuration/_index.md

```diff
@@ -722,6 +722,45 @@ jobs:
 
 This config would update Fedora Rawhide and Fedora 35 dist-git branches.
 
+##### pull_from_upstream
+
+A dist-git only job that opens a new dist-git pull request in
+src.fedoraproject.org when a new upstream release happens using a notification
+from [release-monitoring.org](https://release-monitoring.org/).
+
+This job utilizes the same logic as `propose_downstream` with the only
+exception it is defined and executed in dist-git.
```

Comment:
```suggestion
exception that it is defined and executed in dist-git.
```

Reply:
you're right: either comma is missing or "that", thanks!